cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -13,23 +13,415 @@ License: CC-BY 4.0
+- A description about how to demonstrate that the policy is put in place - how to record it and store it such that an independent
+party can check that the policy exists (version-controlled documents, maintaining a change log, ...)
+- A description on how to prove that the Policy is actually being followed (tying policy statements to evidence)
+- A specification and/or template of the Policy, showing what it means for a policy to foster the development of secure PwDE: 
+for example the list of methodologies of risk assessment, secure by design, secure by default, secure SDLC;
+and a process for effective handling of vulnerabilities by the developers of that product, description of the CVD process
+
+
+### Collaboration with Market Surveillance Authorities
+
+When Market Surveillance Authorities request assistance from a Steward related to one of their projects,
+the Steward shall cooperate with the goal of mitigating security risk posed by the Project they are a Steward for.
+
+In particular, the Market Surveillance Authority may request a copy of the Security Policy.
+
+When an Open Source Project makes public their Security Policy (for example, by publishing on the Project
+site and/or including in the source code repository) then the publication requirement is fulfilled.
+
+However, the regulation also says "in a language which can be easily understood by that authority",
+what might mean the language(s) of the country the market surveillance is located. As most Projects and
+supporting organization publish their policies in English, it would be useful if those authorities accepted
+documents in that language. The language of the country of the Steward may be different than the language
+contributors of a given Project speak or understand.
+
+The Market Surveillance Authority may request documentation related to the Security Policy, with the requirement
+being "put in place and document in a verifiable manner a cybersecurity policy". In addition to the Policy itself,
+it means all other documents showing that the Policy had been applied. It could be for example: the published
+advisory (and in general, a list of published advisories) and the documentation of a CVD process in progress
+(in case of a wide-impact exploited security vulnerability without a published fix, for example).
+
+If a Market Surveillance Authority finds that a Steward does not comply with its obligations (what probably means
+not having the Policy and/or not promoting the best practices), they require Steward to ensure that all appropriate
+corrective actions are taken. As a result, Stewards "shall ensure" (formulation from the law) that all appropriate
+corrective action is taken in respect of their obligations. It means that the Steward needs to have a way to
+influence the Project in question to follow policies. A possible situation may be: not responding to vulnerability
+reports, or not documenting fixed vulnerabilities.
```

Comment:
This is somewhat of a slippery slope. Reporters and developers often disagree on what constitutes a vulnerability, and there are perverse incentives at play which encourage wasting developer time responding to ill-conceived or outright invalid reports. We should not imply that ignoring some reports is necessarily a policy violation, but I suppose that can be an explicit carve-out in whatever policy the project/steward adopts.

Reply:
This is addressed in https://github.com/orcwg/orcwg/pull/206

Please have a look how it looks for you, I'm closing this conversation, let's move it to 206.